uiowa/uiowa
# Hotfix: Hero video and page title color fixes

# Updates
* Load `video-cookies.js` for hero block instead of hero-video-cookie.js
* Updated logic for layout--onecol--background preprocessing to only add `.layout--title--with-background` class when featured image is present.
* Updated style rule to remove unnecessary condition.
* Unrelated reformatting of `video-cookie.js`
* Unrelated comment update.

# How to test

* `blt ds --site=uiowa.edu`
* `drush @home.local uli /`
* Test that pausing and playing the video is persistent through page loads.
* `blt ds --site=sandbox.uiowa.edu`
* `drush @sandbox.local uli about`
* Check page title in Layout Builder
* Add a featured image (and hope your computer doesn't crash!).
* Check that page title looks right on page and in Layout Builder.
